cmd/validate/image.go

```diff
@@ -400,7 +400,7 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 			close(jobs)
 
 			var components []applicationsnapshot.Component
-			var manyData [][]evaluator.Data
+			var evaluatorData [][]evaluator.Data
```

Comment:
Should this be `[]evaludator.Data` instead?

Reply:
Intuitively, I think so, but I don't know if it adds any value and there's an acceptance test for it even, so I'd prefer to leave it alone.